packages/react-components/react-spinbutton/src/stories/SpinButton/SpinButtonAccessibility.md

```diff
@@ -0,0 +1,90 @@
+## Component features and behavior
+
+This section documents accessibility-related behavoirs of spin buttons.
+
+### Keyboarding
+
+#### Tab Order
+1. Value field
+
+#### States
+1. Committed (Focused and value committed)
+2. Editing (Focused and editing)
+
+#### Keyboard State Diagram
+
+| Starting state | Transition | Resulting state |
+| ---------- | ------------ | ------- |
+| Content before spin button | Tab | Committed |
+| Committed  | Tab | Content after spin button (Value committed) |
+| Editing  | Tab | Content after spin button (Value committed) |
+| Editing (Autofill option provided)  | Tab | Content after spin button (Value committed) |
+| Editing  | Enter | Committed |
+| Committed  | Any Edit Key (that results in a change) | Editing |
+| Editing  | Any Edit Key | Editing |
+| Content after spin button (Value committed) | Shift + Tab | Committed |
+
+##### Edit keys
+| Edit Key | Result |
+| ---------- | ------------ |
+| Home | First item in defined range |
+| End | Last item in defined range |
+| Up arrow | One increment higher |
```

Comment:
The `step` prop controls the increment/decrement amount. It defaults to `1` but it could be `10`. What do you think about changing the results cell to

```
`step` increments higher
```

or

```
Increments by small about. Defaults to `1` but can be changed with `step` prop.
```